addons/sale_timesheet/models/project.py

```diff
@@ -372,11 +372,11 @@ def _compute_remaining_hours_so(self):
         for task in self:
             task.remaining_hours_so = mapped_remaining_hours[task._origin.id]
 
-    @api.depends('analytic_account_id.active')
+    @api.depends('so_analytic_account_id.active')
     def _compute_analytic_account_active(self):
         super()._compute_analytic_account_active()
         for task in self:
-            task.analytic_account_active = task.analytic_account_active or task.analytic_account_id.active
+            task.analytic_account_active = task.analytic_account_active or task.so_analytic_account_id.active
```

Comment:
Here I think we don't have a real advantage if we change by one of the two other solutions, but I think it's good to chit-chat about it.

Since `analytic_account_active` should already be in the ORM cache before our for-loop, I think we don't take advantage of `filtered` method. For the second one, we just add 2 lines to have exactly same result, isn't it ?

Reply:
Yeah, it is true, I am agree with you, thanks 🙂